### DISCUSSION TITLE
Speed brake and flaps bug

### DISCUSSION THREAD

**Discussion by lcornar** [flybywiresim/aircraft#5288] [Help] [answered]
Hello, first of all thank you very much for your great work on this project. When loading the plane everything is correct, however, when I place the sim on pause (by ESC key) and resumed it the flaps are full flaps (when before it was full retracted) and the speed brake stop working (It remains at 0% although the lever moves). I use the developer version and I don't know if it happened before the last update ( d5c2c29 ), I use the airbus thrustmaster TCA.

**Comment by CptBayer** [comment, marked as answer]
Hi,

I had this issue as well. But since longer times it is gone ... 

Greets Klaus

**Reply by lcornar** [reply]
I've been reading some forums and the problem is with the TCA, tonight I will try the solution they propose. 
I'd leave it here in case someone who has bought thrustmaster's airbus TCA recently has the same problem. Greetings!!! 

https://forums.flightsimulator.com/t/tca-quadrant-activate-flaps-and-speed-break-when-pressing-esc/372994

**Comment by technopants** [comment]
I have the same problem

**Reply by emsss78** [reply]
Delete the flaps axis binding and any speed brake binding in your TCA Quadrant and it should sort out the problem. I had the same issue and that's what fixed it
